src/ResourceManagement/ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry/project.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "1.1.0-preview",
+  "version": "1.2.0",
```

Comment:
@djyou since you are introducing breaking changes, this will need to remain in preview